pkg/transforms/argoapplication_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+IBM Confidential
+OCO Source Materials
+(C) Copyright IBM Corporation 2019 All Rights Reserved
+The source code for this program is not published or otherwise divested of its trade secrets,
+irrespective of what has been deposited with the U.S. Copyright Office.
+Copyright (c) 2020 Red Hat, Inc.
+*/
+// Copyright Contributors to the Open Cluster Management project
+
+package transforms
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestTransformArgoApplication(t *testing.T) {
+	u := &unstructured.Unstructured{
```

Comment:
This works, but can we read from a json file for consistency with the other resources?